roboflow/roboflowpy.py

```diff
@@ -66,15 +71,83 @@ def upload_image(args):
     )
 
 
+def list_projects(args):
+    rf = roboflow.Roboflow()
+    workspace = rf.workspace(args.workspace)
+    projects = workspace.project_list
+    for p in projects:
+        print()
+        print(p["name"])
+        print(f"  link: {APP_URL}/{p['id']}")
```

Comment:
Did you test for empty (newly created) projects with no versions, images etc.

Reply:
Good call. Did that just now. Working as expected!